build/CopyFilesToStagingDir.ps1

```diff
@@ -55,6 +55,27 @@ PublishFile $FullBuildOutput\RestartAgent\RestartAgent.exe $FullPublishDir\Micro
 PublishFile $FullBuildOutput\RestartAgent\RestartAgent.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
 PublishFile $FullBuildOutput\DeploymentAgent\DeploymentAgent.exe $FullPublishDir\Microsoft.WindowsAppRuntime\
 PublishFile $FullBuildOutput\DeploymentAgent\DeploymentAgent.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+#
+## Publish Kozani Artifacts
+### Kozani WINMDs in MSIX Framework package
+PublishFile $FullBuildOutput\KozaniAppGraph\StrippedWinMD\Microsoft.Kozani.AppGraph.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\StrippedWinMD\Microsoft.Kozani.Package.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\StrippedWinMD\Microsoft.Kozani.SendToLocal.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\StrippedWinMD\Microsoft.Kozani.SendToRemote.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\StrippedWinMD\Microsoft.Kozani.Settings.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniManagerRuntime\StrippedWinMD\Microsoft.Kozani.ManagerRuntime.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+#
+### Kozani DLLs/EXEs in MSIX Framework package
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\KozaniPackage.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\KozaniPackage.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\KozaniSendToLocal.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\KozaniSendToLocal.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\KozaniSendToRemote.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\KozaniSendToRemote.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\KozaniSettings.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\KozaniSettings.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
```

Comment:
Non -winmd artifacts are experimental but I don't see how they're handled differently from non-experimental e.g. KozaniAppGraph.dll is `100% experimental and will be for some time. It should be omitted from preview+stable. How's that indicated in the TransportPackage?